packages/popover/package.json

```diff
@@ -27,9 +27,6 @@
     "@leafygreen-ui/tokens": "^1.3.4",
     "react-transition-group": "^4.4.1"
   },
-  "devDependencies": {
-    "@leafygreen-ui/palette": "^3.4.4"
-  },
```

Comment:
This is one of 2 real changes. Removes unused dev dependency

Reply:
should this be rm'd from the tsconfig as well?